wfe/wfe.go

```diff
@@ -1955,6 +1995,22 @@ func uniqueLowerNames(names []string) []string {
 	return unique
 }
 
+// uniqueIPs returns the set of all unique IP addresses in the input.
+// The returned names will be sorted in assending order.
```

Comment:
I think this comment says "names" when it should be "IP addresses" and there's another small typo here. 

```suggestion
// The returned IP addresses will be sorted in ascending order.
```